src/bin/firebase.js

```diff
@@ -37,27 +38,18 @@ var utils = require("../utils");
 var cmd;
 
 var logFilename = path.join(process.cwd(), "/firebase-debug.log");
-logger
-  .add(winston.transports.Console, {
-    level: process.env.DEBUG ? "debug" : "info",
-    showLevel: false,
-    colorize: true,
-  })
-  .add(winston.transports.File, {
+process.env.DEBUG = _.includes(args, "--debug") || process.env.DEBUG;
```

Comment:
Also worth noting that for tests we have been using `export DEBUG=spec` ... not sure why but we should make sure that still works?

Reply:
that's fair, `process.env.DEBUG` should take precedence over the boolean flag